sample/src/main/kotlin/com/agoda/sample/TestActivity.kt

```diff
@@ -13,11 +14,11 @@ class TestActivity: AppCompatActivity() {
 
     override fun onResume() {
         super.onResume()
-        findViewById(R.id.map).tag = "test_tag"
-        (findViewById(R.id.button) as Button).setOnClickListener {
+        findViewById<ImageView>(R.id.map).tag = "test_tag"
+        (findViewById<Button>(R.id.button)).setOnClickListener {
             startActivity(Intent(this, RecyclerActivity::class.java))
         }
-        (findViewById(R.id.web_button) as Button).setOnClickListener {
+        (findViewById<Button>(R.id.web_button)).setOnClickListener {
```

Comment:
If you remove casting in favor for typed function, remove unnecessary brackets too, please. In all of such cases. Thank you.